src/psyclone/configuration.py

```diff
@@ -574,7 +606,108 @@ def get_default_keys(self):
         return self._config.defaults()
 
 
-class DynConfig(object):
+# =============================================================================
+class APISpecificConfig(object):
+    '''A base class for functions that each API-specific class must provide.
+    At the moment this is just the function 'access_mapping' that maps between
+    API-specific access-descriptor strings and the PSyclone internal 
```

Comment:
pylint complains about trailing white space here.

Reply:
And I just learned that if you use pycharm, and 'go to the end of a line' ... it goes to the last non-space character in a line (i.e. before any trailing white-space) ... strange. Anyway, fixed :)

Reply:
I don't even know what pycharm is :-)